defcon/plugins/test_endpoint.py

```diff
@@ -0,0 +1,30 @@
+"""Test endpoint."""
+from django import test
+
+from defcon.plugins import endpoint
+
+
+class AlertmanagerPluginTests(test.TestCase):
+    """Test the plugins."""
+
+    _API_URLS = [
+        'http://foo.endpoint.io/api/defcon/',
+    ]
+
+    def test_base(self):
+        """Basic test."""
+        p = endpoint.EndpointPlugin()
+        self.assertFalse(p.statuses())
+
+    def test_all(self):
+        """Test with some settings."""
+        for api_url in self._API_URLS:
+            p = endpoint.EndpointPlugin(
```

Comment:
Can you please mock foo.endpoint.io (see examples in other tests)
* once you make it fail (or just use a non-existing hostname, still mock it to avoid network interactions)
* once you make it work

Reply:
two tests
- test_ok: check if all the mandatory fields are here
- test_no_infos: check if the response have all the mandatory, if not send a default payload

i don't have the way to test the 4xx of an url, it's done directly in the endpoint.py

let me know if it's ok